Habitica/src/com/habitrpg/android/habitica/ui/activities/TaskFormActivity.java

```diff
@@ -234,6 +248,13 @@ public void onClick(DialogInterface dialog, int which) {
             createTagsCheckBoxes();
         }
 
+        if (HabiticaApplication.User != null){
+            String allocationMode = HabiticaApplication.User.getPreferences().getAllocationMode();
+            if (TextUtils.isEmpty(allocationMode) || !allocationMode.equals("taskbased")){
+                attributeWrapper.setVisibility(View.GONE);
+            }
+        }
+
```

Comment:
The allocation mode should rather be passed through the intent, like the other task data.